packages/fxa-settings/src/components/Settings/MfaGuard/index.tsx

```diff
@@ -35,6 +36,8 @@ export const MfaGuard = ({
   // Let errors be handled by error boundaries in async contexts
   const handleError = useErrorHandler();
 
+  const hasSentConfirmationCode = useRef(false);
```

Comment:
👍 I'm just going to do a little exploratory testing, but seems like this should work.